packages/truffle-workflow-compile/test/index.js

```diff
@@ -25,30 +36,28 @@ describe("Contracts.compileSources", () => {
 });
 
 describe("Contracts.compile", () => {
-  it("when config.all is true, all contracts in contracts_directory are recompiled", () => {
+  it("when config.all is true, all contracts in contracts_directory are recompiled", async () => {
     let contractName;
 
     // initial compile
-    Contracts.compile(config, (err, { contracts }) => {
+    await Contracts.compile(config, (err, { contracts }) => {
       if (err) assert.fail(err);
```

Comment:
Also, I'm not quite sure what the purpose of this `assert.fail(err)` is.

Reply:
The point is to fail showing `err` as the `AssertionError`, but I like your suggestion the other day to do something more like `assert(false, ${err})`.